Orange/widgets/visualize/tests/test_owsieve.py

```diff
@@ -2,6 +2,8 @@
 # pylint: disable=missing-docstring
 from math import isnan
 from unittest.mock import patch
+
+import Orange
```

Comment:
Don't import `Orange`. `Domain` and `Table` are already imported directly into this file's namespace (see below).

Also, when you do import `Orange`, it has to come after all third-party imports, in this case, it's after `AnyQt`. Otherwise it will trigger one or two pylint warnings (module import order and ungrouped imports).

Reply:
Heh, now I checked what travis/pylint wrote.

```
Orange/widgets/visualize/tests/test_owsieve.py:7: [C0411(wrong-import-order), ] third party import "import numpy as np" should be placed before "import Orange"
Orange/widgets/visualize/tests/test_owsieve.py:12: [C0412(ungrouped-imports), ] Imports from package Orange are not grouped
```

I became a bit of an expert for pylint's messages about this while trying to push a pull request that fixed the imports. :)